exonum-java-binding/core/rust/integration_tests/tests/node_context.rs

```diff
@@ -113,10 +118,10 @@ fn create_node(
     let blockchain = Blockchain::new(
         storage,
         vec![Box::new(EmptyService)],
-        public_key,
-        secret_key,
+        keypair.0,
```

Comment:
why does it take 2 keys instead of one keypair?

Reply:
🤷‍♂️ This is Core API